backend/src/core/config/settings.ts

```
@@ -10,16 +11,38 @@ const envVarsSchema = Joi.object()
       .valid("PRODUCTION", "STAGING", "DEVELOPMENT", "TEST")
       .required(),
     SERVER_PORT: Joi.number().default(8000),
-    JWT_SECRET_KEY: Joi.string().required().description("JWT Secret Key"),
-    BCRYPT_SALT_ROUNDS: Joi.number()
+    // biome-ignore lint/suspicious/noThenProperty: Joi schema requires `then`/`otherwise` shape
+    JWT_SECRET_KEY: Joi.string().when('JWT_ALGORITHM', {
+      is: Joi.string().pattern(/^HS/),
+      then: Joi.required().description("HMAC secret (PEM/base64) for HS* algorithms"),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)